index.js

```diff
@@ -139,14 +144,17 @@ Nanobus.prototype._emit = function (arr, eventName, data, uuid) {
     data = eventName
     eventName = null
   }
+
   var length = arr.length
+  var args = data
```

Comment:
i could alter args in place, but the first clause in the if statement there is pre-pending the eventName.  `shift` tends to be more expensive than just re-binding to a new variable.